composer.json

```diff
@@ -17,7 +17,7 @@
 		"classmap-authoritative": true,
 		"optimize-autoloader": true,
 		"platform": {
-			"php": "7.4"
+			"php": "8.0"
 		},
```

Comment:
When the server bumped the minimum version to >=8, its make necessary to lock the minimum version in all `composer.json` too?

Reply:
I verified the status of this bump in server and don't finished. I think that is good to create an issue to remove the platform section in Talk after the PHP version bump on server will be finished.

Reply:
We can never remove the platform. Each composer file needs to have the platform specified.
But yeah, server is still in the process, as long as 7.4 is not removed, we will not bump here.